src/main/java/jenkins/metrics/impl/TimeInQueueAction.java

```diff
@@ -113,18 +254,88 @@ public long getTotalDurationMillis() {
         return queuingDurationMillis + getBuildingDurationMillis();
     }
 
+    @Exported(visibility = 2)
+    public double getExecutorUtilization() {
+        long buildingDurationMillis = getBuildingDurationMillis();
+        return buildingDurationMillis > 0 ? (getExecutingTimeMillis() * 100 / buildingDurationMillis) / 100.0 : 1.0;
```

Comment:
You probably want to cast one of the operands to double to keep precision?

Reply:
no, i want to force it to 2 decimals ;-)

Reply:
but I should add a comment to that effect